### ISSUE TITLE
Can't connect spkl task runner to CRM Online

### ISSUE THREAD

**Issue by Skubakoob** [scottdurow/SparkleXrm#138]
I'm having trouble getting the spkl task runner connected to crm 365 - if I try to connect to crm6.dynamics.com I get either a "specify username in client credentials" error or an unresolved reference error (if I say no to 365 provisioning). I've tried with/without the org name. Is there a way to give it my credentials?

**Comment by scottdurow** [comment]
You would need to specify a discovery server of crm6.dynamics.com first, select Y to o365 provisioning, then it should prompt you for username/password. Do you not get prompted?
You could also pass a connection string as specified in https://msdn.microsoft.com/en-gb/library/mt608573.aspx

**Comment by Skubakoob** [comment]
HI Scott,

I don't get prompted for the username/password, it just throws the error above. I tried on another machine and it worked no problem at all, so I just copied out the Credentials.xml from C:\Users\<user>\AppData\Roaming\CrmServer to the other machine. I can now select the saved connection and it seems to be working OK. I'll try passing the connection string next time but all working for the moment. Some peculiarity of the other machine I guess!

Anyway, looks a very useful tool so many thanks!

Cheers!
